diff --git a/misc/mc.default.keymap b/misc/mc.default.keymap
index 6e7e8bd..cbcf72d 100644
--- a/misc/mc.default.keymap
+++ b/misc/mc.default.keymap
@@ -1,27 +1,27 @@
 [filemanager]
 ChangePanel = tab; ctrl-i
 Help = f1
-UserMenu = f2
-View = f3
-# ViewFile =
-Edit = f4
+UserMenu = :
+View = V
+# ViewFile = 
+Edit = E
 # EditForceInternal =
-Copy = f5
-Move = f6
-MakeDir = f7
-Delete = f8
+Copy = Y
+Move = M
+MakeDir = N
+Delete = D
 Menu = f9
-Quit = f10
+Quit = esc; Q
 MenuLastSelected = f19
 QuitQuiet = f20
-Find = alt-question
-CdQuick = alt-c
-HotList = ctrl-backslash
+Find = "
+CdQuick = ~
+HotList = `
 Reread = ctrl-r
 DirSize = ctrl-space
 Suspend = ctrl-z
-Swap = ctrl-u
-History = alt-h
+Swap = ctrl-s
+History = H
 # PanelListing =
 # SetupListingFormat =
 ShowHidden = alt-dot
@@ -35,8 +35,8 @@ PutOtherPath = alt-shift-a
 PutCurrentSelected = alt-enter; ctrl-enter
 PutCurrentFullSelected = ctrl-shift-enter
 ViewFiltered = alt-exclamation
-Select = kpplus
-Unselect = kpminus
+Select = ctrl-t
+Unselect = f1
 SelectInvert = kpasterisk
 ScreenList = alt-prime
 # OptionsLayout =
@@ -52,9 +52,9 @@ ScreenList = alt-prime
 # Filter =
 # ConnectShell =
 # ConnectFtp =
-# Undelete =
+Undelete = R
 EditorViewerHistory = alt-shift-e
-ExtendedKeyMap = ctrl-x
+ExtendedKeyMap = X
 
 [filemanager:xmap]
 ChangeMode = c
@@ -67,8 +67,8 @@ LinkSymbolicEdit = ctrl-s
 Link = l
 LinkSymbolic = s
 LinkSymbolicRelative = v
-PanelInfo = i
-PanelQuickView = q
+PanelInfo = q
+PanelQuickView = i
 ExternalPanelize = exclamation
 VfsList = a
 Jobs = j
@@ -81,44 +81,44 @@ PutOtherLink = ctrl-r
 
 [panel]
 CycleListingFormat = alt-t
-Search = ctrl-s; alt-s
-Mark = insert; ctrl-t
+Search = ctrl-n; slash
+Mark = insert; space
 MarkUp = shift-up
 MarkDown = shift-down
 # MarkLeft =
 # MarkRight =
-Down = down; ctrl-n
-Up = up; ctrl-p
-Left = left
-Right = right
-PageUp = pgup; alt-v
-PageDown = pgdn; ctrl-v
+Down = down; j
+Up = up; k
+Left = left; h
+Right = right; l
+PageUp = pgup; ctrl-u
+PageDown = pgdn; ctrl-d
 Enter = enter
-PanelOtherCd = alt-o
-PanelOtherCdLink = alt-l
+PanelOtherCd = alt-l
+PanelOtherCdLink = ctrl-m
 ViewRaw = f13
 EditNew = f14
 CopySingle = f15
 MoveSingle = f16
 DeleteSingle = f18
 # SelectExt =
-Select = alt-plus
-Unselect = alt-minus
-SelectInvert = alt-asterisk
+Select = ctrl-t
+Unselect = f1
+SelectInvert = ctrl-asterisk
 CdChild = ctrl-pgdn
 CdParent = ctrl-pgup
 # CdParentSmart =
-# Panelize =
+Panelize = ctrl-p
 History = alt-shift-h
 HistoryNext = alt-u
 HistoryPrev = alt-y
-BottomOnScreen = alt-j
+BottomOnScreen = ctrl-j
 MiddleOnScreen = alt-r
-TopOnScreen = alt-g
-PanelOtherSync = alt-i
+TopOnScreen = alt-u
+PanelOtherSync = ctrl-l
 SelectCodepage = alt-e
-Top = alt-lt; home; a1
-Bottom = alt-gt; end; c1
+Top = K; home; a1
+Bottom = J; end; c1
 # Sort =
 # SortPrev =
 # SortNext =
@@ -133,11 +133,11 @@ Bottom = alt-gt; end; c1
 
 [dialog]
 Ok = enter
-Cancel = f10; esc; ctrl-g
-Up = left; up
-#Left = left; up
-Down = right; down
-#Right = right; down
+Cancel = f10; esc; ctrl-g; q
+Up = left; k
+#Left = left; h
+Down = right; j
+#Right = right; l
 Help = f1
 Suspend = ctrl-z
 Refresh = ctrl-l
@@ -147,14 +147,14 @@ ScreenPrev = alt-lbrace
 
 [menu]
 Help = f1
-Left = left; ctrl-b
-Right = right; ctrl-f
-Up = up; ctrl-p
-Down = down; ctrl-n
+Left = left; h; ctrl-b
+Right = right; l; ctrl-f
+Up = up; k; ctrl-p
+Down = down; j; ctrl-n
 Home = home; alt-lt; ctr-a
 End = end; alt-gt ctrl-e
 Enter = enter
-Quit = F10; esc; ctrl-g
+Quit = F10; esc; ctrl-g; q
 
 [input]
 Home = ctrl-a; alt-lt; home; a1
@@ -171,13 +171,13 @@ DeleteToWordEnd = alt-d
 Remove = ctrl-w
 # Cut =
 Store = alt-w
-# Paste =
+Paste = ctrl-p
 Yank = ctrl-y
 DeleteToEnd = ctrl-k
 HistoryPrev = alt-p; ctrl-down
 HistoryNext = alt-n; ctrl-up
 History = alt-h
-Complete = alt-tab
+Complete = tab
 # Clear =
 MarkLeft = shift-left
 MarkRight = shift-right
@@ -187,10 +187,10 @@ MarkToHome = shift-home
 MarkToEnd = shift-end
 
 [listbox]
-Up = up; ctrl-p
-Down = down; ctrl-n
-Top = home; alt-lt; a1
-Bottom = end; alt-gt; c1
+Up = up; ctrl-p ; k
+Down = down; ctrl-n; j
+Top = K; alt-lt; a1
+Bottom = J; alt-gt; c1
 PageUp = pgup; alt-v
 PageDown = pgdn; ctrl-v
 Delete = delete; d
@@ -200,10 +200,10 @@ Edit = f4
 Enter = enter
 
 [radio]
-Up = up; ctrl-p
-Down = down; ctrl-n
-Top = home; alt-lt; a1
-Bottom = end; alt-gt; c1
+Up = k; ctrl-p
+Down = j; ctrl-n
+Top = K; alt-lt; a1
+Bottom = J; alt-gt; c1
 Select = space
 
 [tree]
@@ -217,12 +217,12 @@ Up = up; ctrl-p
 Down = down; ctrl-n
 Left = left
 Right = right
-Top = home; alt-lt; a1
-Bottom = end; alt-gt; c1
+Top = K; alt-lt; a1
+Bottom = J; alt-gt; c1
 PageUp = pgup; alt-v
 PageDown = pgdn; ctrl-v
 Enter = enter
-Search = ctrl-s; alt-s
+Search = ctrl-n; slash
 Delete = f8; delete
 
 [help]
@@ -236,8 +236,8 @@ PageDown = f; space; pgdn; ctrl-v
 PageUp = b; pgup; alt-v; backspace
 HalfPageDown = d
 HalfPageUp = u
-Top = home; ctrl-home; ctrl-pgup; a1; alt-lt; g
-Bottom = end; ctrl-end; ctrl-pgdn; c1; alt-gt; shift-g
+Top = K; ctrl-home; ctrl-pgup; a1; alt-lt; g
+Bottom = J; ctrl-end; ctrl-pgdn; c1; alt-gt; shift-g
 Enter = right; enter
 LinkNext = tab
 LinkPrev = alt-tab
@@ -247,8 +247,8 @@ NodePrev = p
 [chattr]
 Up = up; left; ctrl-p
 Down = down; right; ctrl-n
-Top = home; alt-lt; a1
-Bottom = end; alt-gt; c1
+Top = K; alt-lt; a1
+Bottom = J; alt-gt; c1
 PageUp = pgup; alt-v
 PageDown = pgdn; ctrl-v
 Mark = t; shift-t
@@ -275,7 +275,7 @@ End = end
 Tab = tab; shift-tab; ctrl-tab; ctrl-shift-tab
 Undo = ctrl-u
 Redo = alt-r
-Top = ctrl-home; alt-lt
+Top = K; alt-lt
 Bottom = ctrl-end; alt-gt
 ScrollUp = ctrl-up
 ScrollDown = ctrl-down
@@ -294,7 +294,7 @@ EditNew = ctrl-n
 SaveAs = f12; ctrl-f2
 # Close =
 History = alt-shift-e
-Mark = f3
+Mark = space
 Copy = f5
 Move = f6
 Remove = f8
@@ -338,13 +338,13 @@ BlockSave = ctrl-f
 MarkColumn = f13
 Replace = f4
 ReplaceContinue = f14
-Complete = alt-tab
+Complete = tab
 InsertFile = f15
 Quit = f10; esc
 InsertOverwrite = insert
 Help = f1
 # Date =
-Refresh = ctrl-l
+#Refresh = ctrl-l
 Goto = alt-l
 Sort = alt-t
 Mail = alt-m
@@ -409,14 +409,14 @@ Left = h; left
 Right = l; right
 LeftQuick = ctrl-left
 RightQuick = ctrl-right
-Up = k; y; insert; up; ctrl-p
+Up = k; insert; up; ctrl-p
 Down = j; e; delete; down; enter; ctrl-n
 PageDown = f; space; pgdn; ctrl-v
 PageUp = b; pgup; alt-v; backspace
 HalfPageDown = d
 HalfPageUp = u
-Top = home; ctrl-home; ctrl-pgup; a1; alt-lt; g
-Bottom = end; ctrl-end; ctrl-pgdn; c1; alt-gt; shift-g
+Top = K; ctrl-home; ctrl-pgup; a1; alt-lt; g
+Bottom = J; ctrl-end; ctrl-pgdn; c1; alt-gt; shift-g
 BookmarkGoto = m
 Bookmark = r
 FileNext = ctrl-f
@@ -447,12 +447,12 @@ Home = ctrl-a; home
 End = ctrl-e; end
 Left = b; left
 Right = f; right
-Up = k; y; up
+Up = k; up
 Down = j; delete; down
 PageDown = pgdn; ctrl-v
 PageUp = pgup; alt-v
-Top = ctrl-home; ctrl-pgup; a1; alt-lt; g
-Bottom = ctrl-end; ctrl-pgdn; c1; alt-gt; shift-g
+Top = K; ctrl-pgup; a1; alt-lt; g
+Bottom = J; ctrl-pgdn; c1; alt-gt; shift-g
 History = alt-shift-e
 
 [diffviewer]
@@ -466,7 +466,7 @@ Tab2 = 2
 Tab3 = 3
 Tab4 = 4
 Tab8 = 8
-Swap = ctrl-u
+Swap = ctrl-s
 Redo = ctrl-r
 HunkNext = n; enter; space
 HunkPrev = p; backspace
@@ -479,8 +479,8 @@ MergeOther = f15
 Search = f7
 SearchContinue = f17
 Options = f9
-Top = ctrl-home
-Bottom = ctrl-end
+Top = K 
+Bottom = J
 Down = down
 Up = up
 LeftQuick = ctrl-left
diff --git a/misc/mc.ext.ini.in b/misc/mc.ext.ini.in
index c2152ee..4c3665c 100644
--- a/misc/mc.ext.ini.in
+++ b/misc/mc.ext.ini.in
@@ -1031,8 +1031,8 @@ View=%view{ascii} @EXTHELPERSDIR@/doc.sh view ps
 
 [pdf]
 Type=^PDF
-Open=@EXTHELPERSDIR@/doc.sh open pdf
-View=%view{ascii} @EXTHELPERSDIR@/doc.sh view pdf
+Open=zathura %f
+View=zathura %f
 
 # Microsoft Word Document
 [msdoc-by-type]
@@ -1149,11 +1149,12 @@ Open=%cd %p/ucpio://
 View=%view{ascii} @EXTHELPERSDIR@/archive.sh view cpio
 
 [Include/editor]
-Open=%var{EDITOR:vi} %f
+Open=%var{EDITOR:nvim} %f
+Edit=%var{EDITOR:nvim} %f
 
 [Include/image]
-Open=@EXTHELPERSDIR@/image.sh open ALL_FORMATS
-View=%view{ascii} @EXTHELPERSDIR@/image.sh view ALL_FORMATS
+Open=feh %f 
+View=view feh %f ALL_FORMATS
 
 [Include/video]
 Open=@EXTHELPERSDIR@/video.sh open ALL_FORMATS
@@ -1164,7 +1165,8 @@ View=%view{ascii} @EXTHELPERSDIR@/video.sh view ALL_FORMATS
 
 # Default target for anything not described above
 [Default]
-Open=
+Open=%var{EDITOR:nvim} %f
+Edit=%var{EDITOR:nvim} %f
 View=
 
 ### EOF ###
diff --git a/misc/mc.menu.in b/misc/mc.menu.in
index 9d37e37..6891d86 100644
--- a/misc/mc.menu.in
+++ b/misc/mc.menu.in
@@ -1,378 +1,394 @@
 shell_patterns=0
 
 ##############################################################################
-#   %%  The % character
-#   %f  The current file (if non-local vfs, file will be copied locally and
+#   %%  the % character
+#   %f  the current file (if non-local vfs, file will be copied locally and
 #                         %f will be full path to it)
-#   %p  The current file
-#   %d  The current working directory
-#   %s  "Selected files"; the tagged files if any, otherwise the current file
-#   %t  Tagged files
-#   %u  Tagged files (and they are untagged on return from expand_format)
-#   %view Runs the commands and pipes standard output to the view command
-#   If %view is immediately followed by '{', recognize keywords
+#   %p  the current file
+#   %d  the current working directory
+#   %s  "selected files"; the tagged files if any, otherwise the current file
+#   %t  tagged files
+#   %u  tagged files (and they are untagged on return from expand_format)
+#   %view runs the commands and pipes standard output to the view command
+#   if %view is immediately followed by '{', recognize keywords
 #   ascii, hex, nroff and unform
 #
-#   If the format letter is in uppercase, it refers to the other panel
+#   if the format letter is in uppercase, it refers to the other panel
 #
-#   With a number followed the % character you can turn quoting on (default)
-#   and off. For example:
+#   with a number followed the % character you can turn quoting on (default)
+#   and off. for example:
 #   %f    quote expanded macro
 #   %1f   ditto
 #   %0f   don't quote expanded macro
 ##############################################################################
 
 + ! t t
-@       Do something on the current file
-        CMD=%{Enter command}
-        $CMD %f
+@       do something on the current file
+        cmd=%{enter command}
+        $cmd %f
 
 + t t
-@       Do something on the tagged files
-        CMD=%{Enter command}
+@       do something on the tagged files
+        cmd=%{enter command}
         for i in %t ; do
-            $CMD "$i"
+            $cmd "$i"
         done
 
-0       Edit a bug report and send it to root
-        I=`mktemp "${MC_TMPDIR:-/tmp}/mail.XXXXXX"` || exit 1
-        ${EDITOR-vi} "$I"
-        test -r "$I" && mail root < "$I"
-        rm -f "$I"
+# 0       edit a bug report and send it to root
+#         i=`mktemp "${mc_tmpdir:-/tmp}/mail.xxxxxx"` || exit 1
+#         ${editor-vi} "$i"
+#         test -r "$i" && mail root < "$i"
+#         rm -f "$i"
 
-=+ f \.1$ | f \.3$ | f \.4$ | f \.5$ | f \.6$ | f \.7$ | f \.8$ | f \.man$ & t r
-1       Display the file with roff -man
-        %view{ascii,nroff} roff @MAN_FLAGS@ @MANDOC@ %f
-
-2       Call the info hypertext browser
-        info
-
-= t d
-3       Compress the current subdirectory (tar.gz)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | gzip -f9 > "$tar.tar.gz" && \
-        echo "../$tar.tar.gz created."
-
-4       Compress the current subdirectory (tar.bz2)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | bzip2 -f > "$tar.tar.bz2" && \
-        echo "../$tar.tar.bz2 created."
-
-5       Compress the current subdirectory (tar.7z)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | 7za a -si "$tar.tar.7z" && \
-        echo "../$tar.tar.7z created."
-
-6       Compress the current subdirectory (tar.xz)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | xz -f > "$tar.tar.xz" && \
-        echo "../$tar.tar.xz created."
-
-7       Compress the current subdirectory (tar.zst)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | zstd -f > "$tar.tar.zst" && \
-        echo "../$tar.tar.zst created."
-
-8       Compress the current subdirectory (tar.lzo)
-        Pwd=`basename %d /`
-        echo -n "Name of the compressed file (without extension) [$Pwd]: "
-        read tar
-        [ "$tar"x = x ] && tar="$Pwd"
-        cd .. && \
-        tar cf - "$Pwd" | lzop -f > "$tar.tar.lzo" && \
-        echo "../$tar.tar.lzo created."
+# =+ f \.1$ | f \.3$ | f \.4$ | f \.5$ | f \.6$ | f \.7$ | f \.8$ | f \.man$ & t r
+# 1       display the file with roff -man
+#         %view{ascii,nroff} roff  -man %f
+#
+# 2       call the info hypertext browser
+#         info
+#
+# = t d
+# 3       compress the current subdirectory (tar.gz)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | gzip -f9 > "$tar.tar.gz" && \
+#         echo "../$tar.tar.gz created."
+#
+# 4       compress the current subdirectory (tar.bz2)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | bzip2 -f > "$tar.tar.bz2" && \
+#         echo "../$tar.tar.bz2 created."
+#
+# 5       compress the current subdirectory (tar.7z)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | 7za a -si "$tar.tar.7z" && \
+#         echo "../$tar.tar.7z created."
+#
+# 6       compress the current subdirectory (tar.xz)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | xz -f > "$tar.tar.xz" && \
+#         echo "../$tar.tar.xz created."
+#
+# 7       compress the current subdirectory (tar.zst)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | zstd -f > "$tar.tar.zst" && \
+#         echo "../$tar.tar.zst created."
+#
+# 8       compress the current subdirectory (tar.lzo)
+#         pwd=`basename %d /`
+#         echo -n "name of the compressed file (without extension) [$pwd]: "
+#         read tar
+#         [ "$tar"x = x ] && tar="$pwd"
+#         cd .. && \
+#         tar cf - "$pwd" | lzop -f > "$tar.tar.lzo" && \
+#         echo "../$tar.tar.lzo created."
 
 = f \.c$ & t r
 + f \.c$ & t r & ! t t
-c       Compile and link current .c file
-        make "`basename %f .c`" 2>/dev/null || cc -O -o "`basename %f .c`" %f
-
-+ t r & ! t t
-a       Append file to opposite
-        cat %f >> %D/%f
-
-+ t t
-A       Append files to opposite files
-        for i in %t ; do
-            cat "$i" >> %D/"$i"
-        done
-
-+ t r & ! t t
-d       Delete file if a copy exists in the other directory.
-        if [ %d = %D ]; then
-            echo "The two directories must be different."
-            exit 1
-        fi
-        if [ -f %D/%f ]; then        # if two of them, then
-            if cmp -s %D/%f %f; then
-                rm %f && echo %f": DELETED."
-            else
-                echo %f" and "%D/%f" differ: NOT deleted."
-                echo -n "Press RETURN "
-                read key
-            fi
-        else
-            echo %f": No copy in "%D/%f": NOT deleted."
-        fi
+c       compile and link current .c file
+        make "`basename %f .c`" 2>/dev/null || cc -o -o "`basename %f .c`" %f
 
-+ t t
-D       Delete tagged files if a copy exists in the other directory.
-        if [ %d = %D ]; then
-            echo "The two directories must be different."
-            exit 1
+# + t r & ! t t
+# a       append file to opposite
+#         cat %f >> %d/%f
+#
+# + t t
+# a       append files to opposite files
+#         for i in %t ; do
+#             cat "$i" >> %d/"$i"
+#         done
+
+d       diff files with neovim
+        if [ -n "%d/%f" -a -n "%d/%f" ];
+        then nvim -d "%d/%f" "%d/%f";
+        else echo "select two files first!" && read;
         fi
-        for i in %t ; do
-            if [ -f %D/"$i" ]; then
-                SUM1=`sum "$i"`
-                SUM2=`sum %D/"$i"`
-                if [ "$SUM1" = "$SUM2" ]; then
-                    rm "$i" && echo "${i}: DELETED."
-                else
-                    echo "$i and "%D"/$i differ: NOT deleted."
-                fi
-            else
-                echo "$i has no copy in "%D": NOT deleted."
-            fi
-        done
 
-m       View manual page
-        MAN=%{Enter manual name}
-        %view{ascii,nroff} MANROFFOPT='@MAN_FLAGS@' MAN_KEEP_FORMATTING=1 man -P cat "$MAN"
 
-= f \.gz$ & t r
-+ ! t t
-n       Inspect gzip'ed newsbatch file
-        dd if=%f bs=1 skip=12 | zcat | ${PAGER-more}
+# + t t
+# d       diff first tagged files with neovim
+#         active_tagged_files=(%ta);
+#         inactive_tagged_files=(%t);
+#         if [ "${#active_tagged_files[@]}" -gt 0 ] && [ "${#inactive_tagged_files[@]}" -gt 0 ];
+#         then nvim -d "${active_tagged_files[0]}" "${inactive_tagged_files[0]}";
+#         else echo "please tag one file in each panel." && read;
+#         fi
+
+# + t r & ! t t
+# d       delete file if a copy exists in the other directory.
+#         if [ %d = %d ]; then
+#             echo "the two directories must be different."
+#             exit 1
+#         fi
+#         if [ -f %d/%f ]; then        # if two of them, then
+#             if cmp -s %d/%f %f; then
+#                 rm %f && echo %f": deleted."
+#             else
+#                 echo %f" and "%d/%f" differ: not deleted."
+#                 echo -n "press return "
+#                 read key
+#             fi
+#         else
+#             echo %f": no copy in "%d/%f": not deleted."
+#         fi
+#
+# + t t
+# d       delete tagged files if a copy exists in the other directory.
+#         if [ %d = %d ]; then
+#             echo "the two directories must be different."
+#             exit 1
+#         fi
+#         for i in %t ; do
+#             if [ -f %d/"$i" ]; then
+#                 sum1=`sum "$i"`
+#                 sum2=`sum %d/"$i"`
+#                 if [ "$sum1" = "$sum2" ]; then
+#                     rm "$i" && echo "${i}: deleted."
+#                 else
+#                     echo "$i and "%d"/$i differ: not deleted."
+#                 fi
+#             else
+#                 echo "$i has no copy in "%d": not deleted."
+#             fi
+#         done
+
+m       view manual page
+        man=%{enter manual name}
+        %view{ascii,nroff} manroffopt='' man_keep_formatting=1 man -p cat "$man"
+
+# = f \.gz$ & t r
+# + ! t t
+# n       inspect gzip'ed newsbatch file
+#         dd if=%f bs=1 skip=12 | zcat | ${pager-more}
         # assuming the cunbatch header is 12 bytes long.
 
-= t r &
-+ ! t t
-h       Strip headers from current newsarticle
-        CHECK=`awk '{print $1 ; exit}' %f` 2>/dev/null
-        case "$CHECK" in
-            Newsgroups:|Path:)
-                I=`mktemp "${MC_TMPDIR:-/tmp}/news.XXXXXX"` || exit 1
-                cp %f "$I" && sed '/^'"$CHECK"' /,/^$/d' "$I" > %f
-                [ "$?" = "0" ] && rm "$I"
-                echo %f": header removed."
-                ;;
-            *)
-                echo %f" is not a news article."
-                ;;
-        esac
-
-+ t t
-H       Strip headers from the marked newsarticles
-        for i in %t ; do
-            CHECK=`awk '{print $1 ; exit}' "$i"` 2>/dev/null
-            WFILE=`mktemp "${MC_TMPDIR:-/tmp}/news.XXXXXX"` || exit 1
-            case "$CHECK" in
-                Newsgroups:|Path:)
-                    cp "$i" "$WFILE" && sed '/^'"$CHECK"' /,/^$/d' "$WFILE" > "$i"
-                    if [ "$?" = "0" ]; then
-                        rm "$WFILE"; echo "$i header removed. OK."
-                    else
-                        echo "Oops! Please check $i against $WFILE."
-                    fi
-                    ;;
-                *)
-                    echo "$i skipped: Not a news article."
-                    ;;
-            esac
-        done
-
-= t r
-+ ! t t
-r       Copy file to remote host
-        echo -n "To which host?: "
-        read Host
-        echo -n "To which directory on $Host?: "
-        read Dir
-        rcp -p %f "${Host}:${Dir}"
-
-+ t t
-R       Copy files to remote host (no error checking)
-        echo -n "Copy files to which host?: "
-        read Host
-        echo -n "To which directory on $Host? :"
-        read Dir
-        rcp -pr %u "${Host}:${Dir}"
-
-= f \.tex$ & t r
-+ f \.tex$ & t r & ! t t
-t       Run latex on file and show it with xdvi
-        latex %f && xdvi "`basename %f .tex`".dvi
-
-=+ f ^part | f ^Part | f uue & t r
-+ t t
-U       Uudecode marked news articles (needs work)
-        (
-            for i in %t ; do # strip headers
-                FIRST=`awk '{print $1 ; exit}' "$i"`
-                cat "$i" | sed '/^'"$FIRST"' /,/^$/d'
-            done
-        ) | sed '/^$/d' | sed -n '/^begin 6/,/^end$/p' | uudecode
-        if [ "$?" != "0" ]; then
-            echo "Cannot decode "%t"."
-        fi
-        echo "Please test the output file before deleting anything."
-
-=+ f \.tar\.gz$ | f \.tar\.z$ | f \.tgz$ | f \.tpz$ | f \.tar\.lz$ | f \.tar\.lz4$ | f \.tar\.lzma$ | f \.tar\.lzo$ | f \.tar\.7z$ | f \.tar\.xz$ | f \.tar\.zst | f \.tar\.Z$ | f \.tar\.bz2$ & t rl
-x       Extract the contents of a compressed tar file
-        unset PRG
-        case %f in
-            *.tar.7z)   PRG="7za e -so";;
-            *.tar.bz2)  PRG="bunzip2 -c";;
-            *.tar.gz|*.tar.z|*.tgz|*.tpz|*.tar.Z) PRG="gzip -dc";;
-            *.tar.lz)   PRG="lzip -dc";;
-            *.tar.lz4)  PRG="lz4 -dc";;
-            *.tar.lzma) PRG="lzma -dc";;
-            *.tar.lzo)  PRG="lzop -dc";;
-            *.tar.xz)   PRG="xz -dc";;
-            *.tar.zst)  PRG="zstd -dc";;
-            *)          exit 1;;
-        esac
-        $PRG %f | tar xvf -
+# = t r &
+# + ! t t
+# h       strip headers from current newsarticle
+#         check=`awk '{print $1 ; exit}' %f` 2>/dev/null
+#         case "$check" in
+#             newsgroups:|path:)
+#                 i=`mktemp "${mc_tmpdir:-/tmp}/news.xxxxxx"` || exit 1
+#                 cp %f "$i" && sed '/^'"$check"' /,/^$/d' "$i" > %f
+#                 [ "$?" = "0" ] && rm "$i"
+#                 echo %f": header removed."
+#                 ;;
+#             *)
+#                 echo %f" is not a news article."
+#                 ;;
+#         esac
+
+# + t t
+# h       strip headers from the marked newsarticles
+#         for i in %t ; do
+#             check=`awk '{print $1 ; exit}' "$i"` 2>/dev/null
+#             wfile=`mktemp "${mc_tmpdir:-/tmp}/news.xxxxxx"` || exit 1
+#             case "$check" in
+#                 newsgroups:|path:)
+#                     cp "$i" "$wfile" && sed '/^'"$check"' /,/^$/d' "$wfile" > "$i"
+#                     if [ "$?" = "0" ]; then
+#                         rm "$wfile"; echo "$i header removed. ok."
+#                     else
+#                         echo "oops! please check $i against $wfile."
+#                     fi
+#                     ;;
+#                 *)
+#                     echo "$i skipped: not a news article."
+#                     ;;
+#             esac
+#         done
+
+# = t r
+# + ! t t
+# r       copy file to remote host
+#         echo -n "to which host?: "
+#         read host
+#         echo -n "to which directory on $host?: "
+#         read dir
+#         rcp -p %f "${host}:${dir}"
+#
+# + t t
+# r       copy files to remote host (no error checking)
+#         echo -n "copy files to which host?: "
+#         read host
+#         echo -n "to which directory on $host? :"
+#         read dir
+#         rcp -pr %u "${host}:${dir}"
+
+# = f \.tex$ & t r
+# + f \.tex$ & t r & ! t t
+# t       run latex on file and show it with xdvi
+#         latex %f && xdvi "`basename %f .tex`".dvi
+#
+# =+ f ^part | f ^part | f uue & t r
+# + t t
+# u       uudecode marked news articles (needs work)
+#         (
+#             for i in %t ; do # strip headers
+#                 first=`awk '{print $1 ; exit}' "$i"`
+#                 cat "$i" | sed '/^'"$first"' /,/^$/d'
+#             done
+#         ) | sed '/^$/d' | sed -n '/^begin 6/,/^end$/p' | uudecode
+#         if [ "$?" != "0" ]; then
+#             echo "cannot decode "%t"."
+#         fi
+#         echo "please test the output file before deleting anything."
+#
+# =+ f \.tar\.gz$ | f \.tar\.z$ | f \.tgz$ | f \.tpz$ | f \.tar\.lz$ | f \.tar\.lz4$ | f \.tar\.lzma$ | f \.tar\.lzo$ | f \.tar\.7z$ | f \.tar\.xz$ | f \.tar\.zst | f \.tar\.z$ | f \.tar\.bz2$ & t rl
+# x       extract the contents of a compressed tar file
+#         unset prg
+#         case %f in
+#             *.tar.7z)   prg="7za e -so";;
+#             *.tar.bz2)  prg="bunzip2 -c";;
+#             *.tar.gz|*.tar.z|*.tgz|*.tpz|*.tar.z) prg="gzip -dc";;
+#             *.tar.lz)   prg="lzip -dc";;
+#             *.tar.lz4)  prg="lz4 -dc";;
+#             *.tar.lzma) prg="lzma -dc";;
+#             *.tar.lzo)  prg="lzop -dc";;
+#             *.tar.xz)   prg="xz -dc";;
+#             *.tar.zst)  prg="zstd -dc";;
+#             *)          exit 1;;
+#         esac
+#         $prg %f | tar xvf -
 
 = t r
 + ! t t
-y       Gzip or gunzip current file
-        unset DECOMP
-        case %f in
-            *.gz|*.[zZ]) DECOMP=-d;;
-        esac
-        # Do *not* add quotes around $DECOMP!
-        gzip $DECOMP -v %f
-
-+ t t
-Y       Gzip or gunzip tagged files
-        for i in %t ; do
-            unset DECOMP
-            case "$i" in
-                *.gz|*.[zZ]) DECOMP=-d;;
-            esac
-            gzip $DECOMP -v "$i"
-        done
-
-+ ! t t
-b       Bzip2 or bunzip2 current file
-        unset DECOMP
+y       gzip or gunzip current file
+        unset decomp
         case %f in
-            *.bz2) DECOMP=-d;;
+            *.gz|*.[zz]) decomp=-d;;
         esac
-        bzip2 $DECOMP -v %f
+        # do *not* add quotes around $decomp!
+        gzip $decomp -v %f
 
 + t t
-B       Bzip2 or bunzip2 tagged files
+y       gzip or gunzip tagged files
         for i in %t ; do
-            unset DECOMP
+            unset decomp
             case "$i" in
-                *.bz2) DECOMP=-d;;
+                *.gz|*.[zz]) decomp=-d;;
             esac
-            bzip2 $DECOMP -v "$i"
+            gzip $decomp -v "$i"
         done
 
-+ f \.tar.gz$ | f \.tgz$ | f \.tpz$ | f \.tar.Z$ | f \.tar.z$ | f \.tar.bz2$ | f \.tar.F$ & t r & ! t t
-z       Extract compressed tar file to subdirectory
-        unset D
-        set gzip -cd
-        case %f in
-            *.tar.F)   D=`basename %f .tar.F`; set freeze -dc;;
-            *.tar.Z)   D=`basename %f .tar.Z`;;
-            *.tar.bz2) D=`basename %f .tar.bz2`; set bunzip2 -c;;
-            *.tar.gz)  D=`basename %f .tar.gz`;;
-            *.tar.z)   D=`basename %f .tar.z`;;
-            *.tgz)     D=`basename %f .tgz`;;
-            *.tpz)     D=`basename %f .tpz`;;
-        esac
-        mkdir "$D"; cd "$D" && ("$1" "$2" ../%f | tar xvf -)
-
-+ t t
-Z       Extract compressed tar files to subdirectories
-        for i in %t ; do
-            set gzip -dc
-            unset D
-            case "$i" in
-                *.tar.F)   D=`basename "$i" .tar.F`; set freeze -dc;;
-                *.tar.Z)   D=`basename "$i" .tar.Z`;;
-                *.tar.bz2) D=`basename "$i" .tar.bz2`; set bunzip2 -c;;
-                *.tar.gz)  D=`basename "$i" .tar.gz`;;
-                *.tar.z)   D=`basename "$i" .tar.z`;;
-                *.tgz)     D=`basename "$i" .tgz`;;
-                *.tpz)     D=`basename "$i" .tpz`;;
-          esac
-          mkdir "$D"; (cd "$D" && "$1" "$2" "../$i" | tar xvf -)
-        done
-
-+ f \.gz$ | f \.tgz$ | f \.tpz$ | f \.Z$ | f \.z$ | f \.bz2$ & t r & ! t t
-c       Convert gz<->bz2, tar.gz<->tar.bz2 & tgz->tar.bz2
-        unset D
-        unset EXT
-        case %f in
-            *.Z)   EXT=Z;;
-            *.bz2) EXT=bz2;;
-            *.gz)  EXT=gz;;
-            *.tgz) EXT=tgz;;
-            *.tpz) EXT=tpz;;
-            *.z)   EXT=z;;
-        esac
-        case "$EXT" in
-            bz2|Z|gz|z) D=`basename %f ."$EXT"`;;
-            tgz|tpz)    D=`basename %f ."$EXT"`.tar;;
-        esac
-        if [ "$EXT" = "bz2" ]; then
-            bunzip2 -v %f
-            gzip -f9 -v "$D"
-        else
-            gunzip -v %f
-            bzip2 -v "$D"
-        fi
-
-+ t t
-C       Convert gz<->bz2, tar.gz<->tar.bz2 & tgz->tar.bz2
-        for i in %t ; do
-            unset D
-            unset EXT
-            case "$i" in
-                *.Z)   EXT=Z;;
-                *.bz2) EXT=bz2;;
-                *.gz)  EXT=gz;;
-                *.tgz) EXT=tgz;;
-                *.tpz) EXT=tpz;;
-                *.z)   EXT=z;;
-            esac
-            case "$EXT" in
-                bz2|Z|gz|z) D=`basename "$i" ."$EXT"`;;
-                tgz|tpz)    D=`basename "$i" ."$EXT"`.tar;;
-            esac
-            if [ "$EXT" = "bz2" ]; then
-                bunzip2 -v "$i"
-                gzip -f9 -v "$D"
-            else
-                gunzip -v "$i"
-                bzip2 -v "$D"
-            fi
-        done
+# + ! t t
+# b       bzip2 or bunzip2 current file
+#         unset decomp
+#         case %f in
+#             *.bz2) decomp=-d;;
+#         esac
+#         bzip2 $decomp -v %f
+#
+# + t t
+# b       bzip2 or bunzip2 tagged files
+#         for i in %t ; do
+#             unset decomp
+#             case "$i" in
+#                 *.bz2) decomp=-d;;
+#             esac
+#             bzip2 $decomp -v "$i"
+#         done
+
+# + f \.tar.gz$ | f \.tgz$ | f \.tpz$ | f \.tar.z$ | f \.tar.z$ | f \.tar.bz2$ | f \.tar.f$ & t r & ! t t
+# z       extract compressed tar file to subdirectory
+#         unset d
+#         set gzip -cd
+#         case %f in
+#             *.tar.f)   d=`basename %f .tar.f`; set freeze -dc;;
+#             *.tar.z)   d=`basename %f .tar.z`;;
+#             *.tar.bz2) d=`basename %f .tar.bz2`; set bunzip2 -c;;
+#             *.tar.gz)  d=`basename %f .tar.gz`;;
+#             *.tar.z)   d=`basename %f .tar.z`;;
+#             *.tgz)     d=`basename %f .tgz`;;
+#             *.tpz)     d=`basename %f .tpz`;;
+#         esac
+#         mkdir "$d"; cd "$d" && ("$1" "$2" ../%f | tar xvf -)
+
+# + t t
+# z       extract compressed tar files to subdirectories
+#         for i in %t ; do
+#             set gzip -dc
+#             unset d
+#             case "$i" in
+#                 *.tar.f)   d=`basename "$i" .tar.f`; set freeze -dc;;
+#                 *.tar.z)   d=`basename "$i" .tar.z`;;
+#                 *.tar.bz2) d=`basename "$i" .tar.bz2`; set bunzip2 -c;;
+#                 *.tar.gz)  d=`basename "$i" .tar.gz`;;
+#                 *.tar.z)   d=`basename "$i" .tar.z`;;
+#                 *.tgz)     d=`basename "$i" .tgz`;;
+#                 *.tpz)     d=`basename "$i" .tpz`;;
+#           esac
+#           mkdir "$d"; (cd "$d" && "$1" "$2" "../$i" | tar xvf -)
+#         done
+
+# + f \.gz$ | f \.tgz$ | f \.tpz$ | f \.z$ | f \.z$ | f \.bz2$ & t r & ! t t
+# c       convert gz<->bz2, tar.gz<->tar.bz2 & tgz->tar.bz2
+#         unset d
+#         unset ext
+#         case %f in
+#             *.z)   ext=z;;
+#             *.bz2) ext=bz2;;
+#             *.gz)  ext=gz;;
+#             *.tgz) ext=tgz;;
+#             *.tpz) ext=tpz;;
+#             *.z)   ext=z;;
+#         esac
+#         case "$ext" in
+#             bz2|z|gz|z) d=`basename %f ."$ext"`;;
+#             tgz|tpz)    d=`basename %f ."$ext"`.tar;;
+#         esac
+#         if [ "$ext" = "bz2" ]; then
+#             bunzip2 -v %f
+#             gzip -f9 -v "$d"
+#         else
+#             gunzip -v %f
+#             bzip2 -v "$d"
+#         fi
+
+# + t t
+# C       Convert gz<->bz2, tar.gz<->tar.bz2 & tgz->tar.bz2
+#         for i in %t ; do
+#             unset D
+#             unset EXT
+#             case "$i" in
+#                 *.Z)   EXT=Z;;
+#                 *.bz2) EXT=bz2;;
+#                 *.gz)  EXT=gz;;
+#                 *.tgz) EXT=tgz;;
+#                 *.tpz) EXT=tpz;;
+#                 *.z)   EXT=z;;
+#             esac
+#             case "$EXT" in
+#                 bz2|Z|gz|z) D=`basename "$i" ."$EXT"`;;
+#                 tgz|tpz)    D=`basename "$i" ."$EXT"`.tar;;
+#             esac
+#             if [ "$EXT" = "bz2" ]; then
+#                 bunzip2 -v "$i"
+#                 gzip -f9 -v "$D"
+#             else
+#                 gunzip -v "$i"
+#                 bzip2 -v "$D"
+#             fi
+#         done
 
 + x /usr/bin/open | x /usr/local/bin/open & x /bin/sh
 o       Open next a free console
diff --git a/misc/skins/default.ini b/misc/skins/default.ini
index a3dd3e3..74d88ce 100644
--- a/misc/skins/default.ini
+++ b/misc/skins/default.ini
@@ -1,5 +1,5 @@
 [skin]
-    description = Standard skin
+    description = Nice and Dark
 
 [Lines]
     horiz = ─
@@ -25,103 +25,103 @@
     drightmiddle = ┤
 
 [core]
-    _default_ = lightgray;blue
-    selected = black;cyan
-    marked = yellow;blue
-    markselect = yellow;cyan
-    gauge = white;black
-    input = black;cyan
-    inputunchanged = gray;cyan
-    inputmark = cyan;black
-    disabled = gray;lightgray
-    reverse = black;lightgray
+    _default_ = lightgray;black
+    selected = black;green
+    marked = brown;black
+    markselect = brown;blue
+    gauge = black;lightgray
+    input = lightgray;black
+    inputunchanged = gray;black
+    inputmark = black;lightgray
+    disabled = gray;black
+    reverse = brown;black
     commandlinemark = black;lightgray
-    header = yellow;blue
+    header = lightgray;black
     inputhistory =
     commandhistory =
     shadow = gray;black
 
 [dialog]
-    _default_ = black;lightgray
-    dfocus = black;cyan
-    dhotnormal = blue;lightgray
-    dhotfocus = blue;cyan
-    dtitle = blue;lightgray
+    _default_ = lightgray;black
+    dfocus = lightgray;blue
+    dhotnormal = brown;black
+    dhotfocus = brown;blue
+    dtitle = brown;black
 
 [error]
-    _default_ = white;red
-    errdfocus = black;lightgray
-    errdhotnormal = yellow;red
-    errdhotfocus = yellow;lightgray
-    errdtitle = yellow;red
+    _default_ = red;black
+    errdfocus = black;red
+    errdhotnormal = brightred;black
+    errdhotfocus = brown;red
+    errdtitle = brown;black
 
 [filehighlight]
-    directory = white;
+    directory = blue;
     executable = brightgreen;
-    symlink = lightgray;
+    symlink = gray;
     hardlink =
     stalelink = brightred;
     device = brightmagenta;
-    special = black;
+    special = gray;
     core = red;
     temp = gray;
-    archive = brightmagenta;
-    doc = brown;
-    source = cyan;
+    archive = magenta;
+    doc = lightgray;
+    source = brightcyan;
     media = green;
-    graph = brightcyan;
+    graph = cyan;
     database = brightred;
 
 [menu]
-    _default_ = white;cyan
-    menusel = white;black
-    menuhot = yellow;cyan
-    menuhotsel = yellow;black
-    menuinactive = black;cyan
+    _default_ = lightgray;black
+    menusel = lightgray;blue
+    menuhot = brown;black
+    menuhotsel = brown;blue
+    menuinactive = gray;black
 
 [popupmenu]
-    _default_ = white;cyan
-    menusel = yellow;black
-    menutitle = yellow;cyan
+    _default_ = lightgray;black
+    menusel = lightgray;blue
+    menutitle = lightgray;black
 
 [buttonbar]
-    hotkey = white;black
-    button = black;cyan
+    hotkey = lightgray;blue
+    button = lightgray;black
 
 [statusbar]
-    _default_ = black;cyan
+    _default_ = lightgray;black
 
 [help]
-    _default_ = black;lightgray
-    helpitalic = red;lightgray
-    helpbold = blue;lightgray
-    helplink = black;cyan
-    helpslink = yellow;blue
-    helptitle = blue;lightgray
+    _default_ = lightgray;black
+    helpitalic = red;black
+    helpbold = brown;black
+    helplink = blue;black
+    helpslink = lightgray;blue
+    helptitle = brown;black
 
 [editor]
-    _default_ = lightgray;blue
-    editbold = yellow;green
-    editmarked = black;cyan
+    _default_ = lightgray;black
+    editbold = brown;blue
+    editmarked = grey;cyan
     editwhitespace = brightblue;blue
-    editnonprintable = ;black
+    editnonprintable = grey;black
     editlinestate = white;cyan
     bookmark = white;red
-    bookmarkfound = black;green
+    bookmarkfound = grey;green
     editrightmargin = brightblue;black
-#    editbg = lightgray;
-#    editframe = lightgray;
-    editframeactive = white;
-    editframedrag = green;
+#    editbg =
+    editframe = gray;
+    editframeactive = lightgray;
+    editframedrag = brightblue;
 
 [viewer]
-    _default_ = lightgray;blue
-    viewbold = yellow;blue
+    _default_ = lightgray;black
+    viewbold = brown;blue
     viewunderline = brightred;blue
-    viewselected = yellow;cyan
+    viewselected = brown;cyan
 
 [diffviewer]
-    added = white;green
+    added = white;brown
     changedline = blue;cyan
     changednew = red;cyan
     changed = white;cyan
@@ -129,11 +129,16 @@
     error = red;lightgray
 
 [widget-panel]
-    sort-up-char = '
-    sort-down-char = .
-    filename-scroll-left-char = {
-    filename-scroll-right-char = }
+    sort-up-char = ↑
+    sort-down-char = ↓
+    hiddenfiles-show-char = •
+    hiddenfiles-hide-char = ○
+    history-prev-item-char = «
+    history-next-item-char = »
+    history-show-list-char = ^
+    filename-scroll-left-char = «
+    filename-scroll-right-char = »
 
 [widget-editor]
-    window-state-char = *
-    window-close-char = X
+    window-state-char = ↕
+    window-close-char = ✕
diff --git a/src/filemanager/panel.c b/src/filemanager/panel.c
index f14fd38..8b2b104 100644
--- a/src/filemanager/panel.c
+++ b/src/filemanager/panel.c
@@ -2808,8 +2808,11 @@ do_search (WPanel * panel, int c_code)
             }
         }
     }
-
+    if (panel->quick_search.buffer->len == 0 || panel->quick_search.buffer->str[0] != '*') {
+        g_string_prepend(panel->quick_search.buffer, "*");
+    }
     reg_exp = g_strdup_printf ("%s*", panel->quick_search.buffer->str);
+
     esc_str = strutils_escape (reg_exp, -1, ",|\\{}[]", TRUE);
     search = mc_search_new (esc_str, NULL);
     search->search_type = MC_SEARCH_T_GLOB;
diff --git a/src/setup.c b/src/setup.c
index 9d6e153..1c5865a 100644
--- a/src/setup.c
+++ b/src/setup.c
@@ -82,9 +82,9 @@
 gboolean boot_current_is_left = TRUE;
 
 /* If on, default for "No" in delete operations */
-gboolean safe_delete = FALSE;
+gboolean safe_delete = TRUE;
 /* If on, default for "No" in overwrite files */
-gboolean safe_overwrite = FALSE;
+gboolean safe_overwrite = TRUE;
 
 /* Controls screen clearing before an exec */
 gboolean clear_before_exec = TRUE;
@@ -135,13 +135,14 @@ panels_options_t panels_options = {
     .mark_moves_down = TRUE,
     .reverse_files_only = TRUE,
     .auto_save_setup = FALSE,
-    .navigate_with_arrows = FALSE,
+    .navigate_with_arrows = TRUE,
     .scroll_pages = TRUE,
-    .scroll_center = FALSE,
-    .mouse_move_pages = TRUE,
+    .scroll_center = TRUE,
+    .mouse_move_pages = FALSE,
     .filetype_mode = TRUE,
     .permission_mode = FALSE,
-    .qsearch_mode = QSEARCH_PANEL_CASE,
+    .qsearch_mode = QSEARCH_CASE_INSENSITIVE,
+    // .qsearch_mode = 0,
     .torben_fj_mode = FALSE,
     .select_flags = SELECT_MATCH_CASE | SELECT_SHELL_PATTERNS
 };
@@ -149,7 +150,7 @@ panels_options_t panels_options = {
 gboolean easy_patterns = TRUE;
 
 /* It true saves the setup when quitting */
-gboolean auto_save_setup = TRUE;
+gboolean auto_save_setup = FALSE;
 
 /* If true, then the +, - and \ keys have their special meaning only if the
  * command line is empty, otherwise they behave like regular letters
@@ -157,10 +158,10 @@ gboolean auto_save_setup = TRUE;
 gboolean only_leading_plus_minus = TRUE;
 
 /* Automatically fills name with current selected item name on mkdir */
-gboolean auto_fill_mkdir_name = TRUE;
+gboolean auto_fill_mkdir_name = FALSE;
 
 /* If set and you don't have subshell support, then C-o will give you a shell */
-gboolean output_starts_shell = FALSE;
+gboolean output_starts_shell = TRUE;
 
 #ifdef USE_FILE_CMD
 /* If set, we execute the file command to check the file type */
@@ -177,9 +178,9 @@ gboolean verbose = TRUE;
 gboolean file_op_compute_totals = TRUE;
 
 /* If true use the internal viewer */
-gboolean use_internal_view = TRUE;
+gboolean use_internal_view = FALSE;
 /* If set, use the builtin editor */
-gboolean use_internal_edit = TRUE;
+gboolean use_internal_edit = FALSE;
 
 #ifdef HAVE_CHARSET
 /* Numbers of (file I/O) and (input/display) codepages. -1 if not selected */
@@ -698,6 +699,7 @@ panels_load_options (void)
 
         qmode = mc_config_get_int (mc_global.main_config, CONFIG_PANELS_SECTION,
                                    "quick_search_mode", (int) panels_options.qsearch_mode);
+        qmode = -1;
         if (qmode < 0)
             panels_options.qsearch_mode = QSEARCH_CASE_INSENSITIVE;
         else if (qmode >= QSEARCH_NUM)
